tests/e2e/controllers/messages.js

```diff
@@ -0,0 +1,208 @@
+const assert = require('chai').assert,
+  PouchDB = require('pouchdb'),
+  request = require('request-promise-native'),
+  Url = require('url'),
+  utils = require('../utils');
+
+const adminDb = new PouchDB(process.env.COUCH_URL);
+
+const pendingId = 'ff827d04-7083-4ff3-e53807a03a54d8af';
+
+const data_record = {
+  _id: 'my_data_record',
+  errors: [],
+  form: null,
+  from: '0211111111',
+  reported_date: 1432801258088,
+  tasks: [
+   {
+       state: 'sent',
```

Comment:
Nitpick: use 2 spaces for indentation not 4. Depending on your tool there is probably an easy way to convert this.